var/spack/repos/builtin/packages/libgcrypt/package.py

```diff
@@ -32,8 +32,10 @@ class Libgcrypt(AutotoolsPackage):
        key algorithms, large integer functions, random numbers and a lot
        of supporting functions. """
     homepage = "http://www.gnu.org/software/libgcrypt/"
-    url      = "http://gd.tuwien.ac.at/pub/gnupg/libgcrypt/libgcrypt-1.6.2.tar.bz2"
+    url 	 = "http://gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.7.6.tar.bz2"
```

Comment:
Sorry for the confusion. That is only the case for `openssl` (and anything that it depends on). The reason for this is that in order to download from `https`, you need a working `openssl`. So if you don't have `openssl` on your system, using `https` would make it impossible to install. For every other package, `https` is probably preferred when possible. I assume that was the original reason for this PR?